client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
```

Comment:
I don't know exactly what you did, but it'd be fine to just change line 176, that is, to still use a local variable `elem` in the `start` method and then assign the value of that variable to `this.elem` as well.